style.css

```diff
@@ -751,34 +600,938 @@ input[type="email"]::placeholder {
   height: 48px;
   font-size: 17px;
   border-radius: 8px;
-  float: left;
-  margin-top: 22px;
-}
-
-.contact-b .button:hover {
-  text-decoration: none;
-  background-color: var(--color-main);
-  align-self: flex-start;
-  color: white;
-  width: 140px;
-  height: 48px;
-  font-size: 17px;
-  border-radius: 8px;
-  float: left;
   margin-top: 22px;
-  border: 1px solid white;
 }
 
-.contact-b .button:active {
-  text-decoration: none;
-  background-color: var(--button-pressed);
-  align-self: flex-start;
-  color: white;
-  width: 140px;
-  height: 48px;
-  font-size: 17px;
-  border-radius: 8px;
-  float: left;
-  margin-top: 22px;
-  border: 1px solid white;
+@media only screen and (max-width: 768px) {
+  #projects {
+    width: 100%;
+    height: 350vh;
```

Comment:
- [ ] Please use media query as specified in the project requirement  `Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).` to style for the desktop screen.

- Hint: `@media screen and (min-width: 768px){  }` or wider.
All styles implemented in that range will be in desktop screen sizes 👍